qa/rpc-tests/test_framework/zip244.py

```diff
@@ -177,7 +177,7 @@ def orchard_actions_noncompact_digest(orchardBundle):
     for desc in orchardBundle.actions:
         digest.update(ser_uint256(desc.cv))
         digest.update(ser_uint256(desc.rk))
-        digest.update(desc.encCiphertext[564:])
+        digest.update(desc.encCiphertext[52+564:])
```

Comment:
Please check this.

Reply:
Ditto; the original code should be correct, i.e. `52 + 512`